crates/rpc-types-debug/src/debug.rs

```diff
@@ -1,25 +1,27 @@
 //! Types for the `debug` API.
 
 use alloc::{collections::btree_map::BTreeMap, vec::Vec};
-use alloy_primitives::{Bytes, StorageKey, StorageValue, B256};
+use alloy_primitives::{Bytes, StorageKey, B256};
 use serde::{Deserialize, Serialize};
 
 /// Represents the result of a storage slot query.
 #[derive(Clone, Debug, Default, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
 pub struct StorageResult {
-    /// The key of the storage slot
-    pub key: B256,
     /// The storage key
-    pub storage_key: StorageKey,
+    pub key: StorageKey,
     /// The value stored at the slot
-    pub value: StorageValue,
+    pub value: B256,
 }
 
-/// Represents a map of storage slots.
-pub type StorageMap = BTreeMap<B256, StorageResult>;
+/// Wrapper type for a map of storage slots.
+#[derive(Clone, Debug, Default, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct StorageMap(pub BTreeMap<B256, StorageResult>);
```

Comment:
we can also derive asref asmut and refer for this

like we do here

https://github.com/alloy-rs/alloy/blob/ff758f86622c343457c4fe856a88033819f11d6e/crates/consensus/src/transaction/recovered.rs#L11-L11
if this isnt a dep, we can add

https://github.com/alloy-rs/alloy/blob/ff758f86622c343457c4fe856a88033819f11d6e/crates/consensus/src/transaction/recovered.rs#L8-L8